scripts/zones/VeLugannon_Palace/mobs/Zipacna.lua

```diff
@@ -4,122 +4,523 @@
+local ID = zones[xi.zone.VELUGANNON_PALACE]
 
-local pathNodes =
+local firstBlueDoorCoord = { x = -160, y = -2.974, z = 380 }
+local middleBlueDoorCoord = { x = -80, y = -2.974, z = 460 }
+local middleYellowDoorCoord = { x = 80, y = -2.974, z = 460 }
+local firstYellowDoorCoord = { x = 160, y = -2.974, z = 380 }
+
+-----------------------------------
+---Doors are named from West to East
+---1st Blue Door | Middle Blue Door | Middle Yellow Door | 1st Yellow Door
+-----------------------------------
+
+    -- From blue side spawn point to the first blue door
+local blueShortSideSpawnPath =
+{
+    { x = -195, z = 395 },
+    { x = -192, z = 390 },
+    { x = -189, z = 384 },
+    { x = -183, z = 380 },
+    { x = -165, z = 380 },
+    { x = -162, z = 380 },
+}
+
+    -- From the blue side basement to the first blue door
+local blueBasementSpawnPath =
+{
+    { x = -134, y = 16, z = 380 },
+    { x = -150, y = 16, z = 379 },
+    { x = -220, y = 16, z = 379 },
+    { x = -220, y = 16, z = 416 },
+    { x = -232, y = 16, z = 419 },
+    { x = -240, y = 16, z = 419 }, -- bottom of steps going back upstairs
+    { x = -255, y = 12, z = 418 },
+    { x = -260, y = 11, z = 412 },
+    { x = -260, y = 8, z = 401 },
+    { x = -259, y = 5, z = 388 },
+    { x = -255, y = 3, z = 380 },
+    { x = -239, y = 0, z = 380 },
+    { x = -215, y = 0, z = 380 },
+    { x = -204, y = 0, z = 389 },
+    { x = -197, y = 0, z = 391 },
+    { x = -186, y = 0, z = 382 },
+    { x = -177, y = 0, z = 380 },
+    { x = -162, y = 0, z = 380 }, --first blue door
+}
+
+    -- From the short yellow side spawn point to the first yellow door
+local yellowShortSideSpawnPath =
+{
+    { x = 191, y = 0, z = 399 },
+    { x = 192, y = 0, z = 389 },
+    { x = 186, y = 0, z = 381 },
+    { x = 182, y = 0, z = 379 },
+    { x = 174, y = 0, z = 379 },
+    { x = 161, y = 0, z = 379 },
+}
+
+    -- From the yellow basement to the first yellow door
+local yellowBasementSpawnPath =
+{
+    { x = 173, y = 16, z = 458 },
+    { x = 178, y = 16, z = 460 },
+    { x = 210, y = 16, z = 460 },
+    { x = 215, y = 16, z = 457 },
+    { x = 219, y = 16, z = 452 },
+    { x = 219, y = 16, z = 442 },
+    { x = 219, y = 16, z = 420 },
+    { x = 240, y = 16, z = 419 },
+    { x = 258, y = 12, z = 417 },
+    { x = 259, y = 4, z = 384 },
+    { x = 254, y = 3, z = 380 },
+    { x = 239, y = 0, z = 379 },
+    { x = 214, y = 0, z = 380 },
+    { x = 208, y = 0, z = 386 },
+    { x = 208, y = 0, z = 398 },
+    { x = 206, y = 0, z = 405 },
+    { x = 198, y = 0, z = 409 },
+    { x = 191, y = 0, z = 405 },
+    { x = 192, y = 0, z = 395 },
+    { x = 193, y = 0, z = 389 },
+    { x = 187, y = 0, z = 382 },
+    { x = 184, y = 0, z = 380 },
+    { x = 176, y = 0, z = 379 },
+    { x = 157, y = 0, z = 380 },
+}
+
+    -- From first blue door to the blue basement loop
+local firstBlueToBasement =
+{
+    { x = -162, y = 0, z = 380 }, --First Blue Door
+    { x = -171, y = 0, z = 379 },
```

Comment:
Comment formatting:
```lua
-- From the first blue door to the blue basement loop
local firstBlueToBasement =
{
    { x = -162, y = 0, z = 380 }, -- First Blue Door
    { x = -171, y = 0, z = 379 },
```

Reply:
Helping Hook out with this - So will comment on some of this on his behalf.
These should all be fixed now I believe.